normandy/recipes/tests/test_commands.py

```diff
@@ -522,48 +498,32 @@ def test_publishes_missing_recipes(self, rs_settings, requestsmock):
         # The first two requests should be to get the existing records
```

Comment:
The comments in this section refer to pairs of requests ("the first two") which was true when we were interacting with both collections, but should be updated now to only talk about one request.